### ISSUE TITLE
Can't move minecarts using movement keys

### ISSUE THREAD

**Issue by maxutka99** [gnembon/fabric-carpet#1680]
Nothing happens when hold W in a minecart. It should start moving but it doesn't. All carpet rules are disabled.

Steps to reproduce:

1. Join singleplayer world with carpet mod installed
2. Place rails and install minecart on it
3. Sit in minecart and hold W

**Comment by altrisi** [comment]
Probably related to the refactor to riding in 1.19.4, can't find what in Carpet yet though.

**Comment by altrisi** [comment]
Seems to be caused by (at least) one of these mixins:
```json
    "Player_fakePlayersMixin",
    "ServerGamePacketListenerImpl_coreMixin",
    "ServerGamePacketListenerImpl_tickMixin",
    "MinecraftServer_scarpetMixin",
    "ExperienceOrb_xpNoCooldownMixin",
    "Player_xpNoCooldownMixin",
    "ServerPlayer_actionPackMixin",
```

**Comment by altrisi** [comment]
One of:
```json
    "ServerGamePacketListenerImpl_coreMixin",
    "ServerGamePacketListenerImpl_tickMixin",
    "ServerPlayer_actionPackMixin"
```

**Comment by altrisi** [comment]
Fixed in 1.4.101
